flask-task-monitoring/flask_task_monitoring/api.py

```diff
@@ -26,7 +26,7 @@ async def ask_question():
     # the Prefect API.  Instead, they are stored in the common result storage area that
     # the application and task servers share (in this example application, that is a
     # filesystem path).
-    answer: TaskRun = get_help.apply_async(args=(question,))
+    answer: TaskRun = get_help.delay(question)
```

Comment:
This type hint is out of date, these methods now return a future. More of a reminder to myself etc